apps/vite/src/components/UserSettings.tsx

```
@@ -1,45 +1,46 @@
+  photoUrl?: string | null;
+  status: string;
 }) => {
-  const { t, i18n } = useTranslation()
-  const { toggleDarkMode, darkMode } = useContext(DarkModeContext)
+  const { t, i18n } = useTranslation();
+  const { toggleDarkMode, darkMode } = useContext(DarkModeContext);
   return (
     <Menu as="div" className="relative ml-3">
       <div className="flex items-center gap-1 rounded-full border p-0.5 shadow-sm dark:border-gray-400 ">
-        {status != "NOT_LOGGED_IN" && (
+        {status != 'NOT_LOGGED_IN' && (
```

Biome

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)